packages/jest-cli/src/TestRunner.js

```diff
@@ -289,9 +323,40 @@ const addResult = (
   } else {
     aggregatedResults.numPassedTestSuites++;
   }
+
+  // Snapshot data
+
+
+  if (testResult.snapshot.added) {
+    aggregatedResults.snapshot.filesAdded++;
```

Comment:
i copied it from the reporter, but i'm pretty sure that it should be
```js
aggregatedResults.snapshot.filesAdded += testResult.snapshot.added;
```
because `testResult.snapshot.added` is a number in type definition.
however, all tests pass with the current implementation